src/templates/Component.ts

```diff
@@ -0,0 +1,23 @@
+import { DNode, WidgetProperties } from '@dojo/widget-core/interfaces';
+import { WidgetBase } from '@dojo/widget-core/WidgetBase';
+import { ThemeableMixin, theme } from '@dojo/widget-core/mixins/Themeable';
+
+import * as css from './<%= componentStylePath %>';
+
+/**
+ * @type <%= name %>Properties
+ *
+ * Properties that can be set on <%= name %> components
+ */
+export interface <%= name %>Properties extends WidgetProperties { };
```

Comment:
Also don't have to extend `WidgetProperties` either (they are implicitly available)